crucible-llvm/src/Lang/Crucible/LLVM/MemModel.hs

```diff
@@ -83,9 +83,12 @@ module Lang.Crucible.LLVM.MemModel
   , doArrayStoreUnbounded
   , doArrayConstStore
   , doArrayConstStoreUnbounded
-  , loadString
-  , loadMaybeString
-  , strLen
+  , {-# DEPRECATED "Exported from Crucible.LLVM.MemModel.Strings instead" #-}
+    loadString
+  , {-# DEPRECATED "Exported from Crucible.LLVM.MemModel.Strings instead" #-}
+    loadMaybeString
+  , {-# DEPRECATED "Exported from Crucible.LLVM.MemModel.Strings instead" #-}
+    strLen
```

Comment:
NTS: Deprecating exports is only available in GHC 9.8, these deprecations will have to wait for #1308.